plugins/fusion/plugin.py

```diff
@@ -274,6 +274,15 @@ def __init__(self, *args, **kwargs):
 
         self.remote_donation_address: str = ''  # optionally announced by the remote server in 'serverhello' message
 
+        if tuple(self.config.get('cashfusion_server', ())) == ('cashfusion.electroncash.dk', 8787, False):
+            # User's config has the old default non-SSL server. If we see this,
+            # just wipe the config key so that the new default is used.
+            # But only reset once, after that let them go back if that is what
+            # they truly desire.
+            if self.config.get('cashfusion_server_defaultresetted', 0) < 1:
```

Comment:
@cculianu I'm embarrassed by this, please feel free to yell at me if you think there's a better way.

Reply:
Eh. We have to do this sometimes.  There are other places in the code where similar transition/fixup logic occurs. This is fine.